src/main/java/github/scarsz/discordsrv/util/WebhookUtil.java

```diff
@@ -113,67 +118,45 @@ public static void deliverMessage(TextChannel channel, Player player, String mes
         });
     }
 
-    static class LastWebhookInfo {
-
-        final String webhook;
-        final String targetName;
+    private static final Map<TextChannel, List<Webhook>> channelWebhooks = new ConcurrentHashMap<>();
+    private static final Map<TextChannel, Webhook> lastUsedWebhooks = new ConcurrentHashMap<>();
```

Comment:
Storing JDA entities is generally a bad idea, they tend to die

Reply:
Oh well, it was fun while it lasted https://github.com/DV8FromTheWorld/JDA/blob/master/README.md#entity-lifetimes